### PR TITLE
Reviewing naming of Variable Sets

### PR DESCRIPTION
**Pull request by MarkSiedle**
In preparation for Vertical Navigation's release next week, we've been reviewing docs and discovered various combinations of the wording around "Variable Sets".

This PR reviews the various wording and standardises where appropriate, based on both the current UI and the new UI coming soon. We no longer refer to "Variable Sets" as "Library Variable Sets" or "Library Sets". They are just Variable Sets.

### CODE REVIEW

**Comment by steve-fenton-octopus** [comment]
There are quite a few images in here with a missing opening `(`...

```
![]/docs/shared-content/octopus-recommendations/configure-project/images/projectconfiguration-deployareleasemanualintervention.png)
```

Should be:

```
![](/docs/shared-content/octopus-recommendations/configure-project/images/projectconfiguration-deployareleasemanualintervention.png)
```

Sending a commit to fix this and resolve some spelling issues.

**Comment by steve-fenton-octopus** [comment]
I can sort out this merge for you tomorrow.

**Comment by MarkSiedle** [comment]
Ah it was the image tags! I was so confused by the spell check errors. Thank you. I'm still reviewing some of these changes later today, but this is in a much nicer position now, thank you.

**Comment by MarkSiedle** [comment]
Thanks @steve-fenton-octopus. This one is now ready for review please 🙏

**Review by steve-fenton-octopus — APPROVED** [review]
LGTM!

I'll add a note to the "next edition feedback" of Exploring Octopus Deploy to also remove "library" and just use "variable set".

File: src/pages/docs/support/debug-problems-with-octopus-variables.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ navOrder: 1
 
 Sometimes a variable used during deployment may have a different value from the one you expect. Here are the first steps to debugging these issues.
 
-## Check the variable snapshot for the release {#DebugproblemswithOctopusvariables-ChecktheVariableSnapshotfortheRelease}
+## Check the variable snapshot for the release {#check-variable-snapshot-for-release}
```
Comment (MarkSiedle) [inline src/pages/docs/support/debug-problems-with-octopus-variables.md:12]: I followed the same approach Steve Fenton used in this previous PR where we encountered this: https://github.com/OctopusDeploy/docs/pull/2360/commits/2bd1155bb6a346f4ff1819565f39c3bc515f9a70

And confirmed there's no anchor link references to these that needed updating.

**Pull request merged**